eventlet/support/greendns.py

```diff
@@ -179,11 +179,16 @@ def _readlines(self):
         Note that this performs disk I/O so can be blocking.
         """
         lines = []
+        if sys.version_info < (3, 4):
+            mode = 'rU'
+        else:
+            mode = 'r'
```

Comment:
'rb' means that everything that was being done 'magically' (and in C, i.e. faster) by `open` has to be done explicitly. However, if that's what you want, you can use #426 instead.